index.html

```diff
@@ -122,6 +139,302 @@ <h5 class="color-purple-mountains-majesty" localization="why-myriad-distribution
       </div>
     </div>
   </div>
+  
+  
+  
+
+<!-- new code for members page >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> -->
+
+
+<div id="Members" class="section section-2 row">
+    <div class="col-xs-10 col-xs-push-1 col-sm-8 col-sm-push-2 col-md-6 col-md-push-3 text-center">
+      <h1 class="title" localization="wallets-section-header">Members</h1>
+	  
+    </div>
+
+    <div class="col-xs-12 col-sm-10 col-sm-push-1">
+      <div class="panel">
+        <div class="panel-body">
+          <div class="row">
+            <div class="col-xs-12 col-sm-3">
+              <div class="info-group text-left">
+                <h5 localization="wallets-desktop-title">Coders</h5>
+
+                <div class="platform">
+<p>
+<b>  8bitcoder - Creator</b> : 
+<p>.................................... </p>
+<p>
+<b>cryptapus -  Developer </b>
+<p>.................................... </p>
+<p>
+<b>Soft_Toilet_Paper</b>
+<p>.................................... </p>
+<p>
+<b>_WLC_</b>
+<p>.................................... </p>
+<p>
+<b>roarde</b>
+<p>___________________________</p>
+ </div>
```

Comment:
I've attempted to keep a listing of who's contributed to the core code here:

https://github.com/myriadteam/myriadcoin/blob/master/doc/release-notes.md#myriadcoin-credits

Unsure if this information should be included on the website, Ideally Myriadcoin shouldn't have a central authority... leaning towards NAK on this but won't block.

Reply:
i think for marketing it would be great to have names on website, but if most people dont want it, than it could be removed